packages/@aws-cdk/aws-dynamodb/test/dynamodb.test.ts

```diff
@@ -363,6 +365,7 @@ testDeprecated('when specifying every property', () => {
           'Fn::GetAtt': ['MyStream5C050E93', 'Arn'],
         },
       },
+      TableClass: TableClass.STANDARD,
```

Comment:
Since we're testing this explicitly anyway below, let's leave this existing test unchanged.